xbmc/windowing/X11/WinSystemX11GLESContext.h

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "WinSystemX11.h"
```

Comment:
wrong include order and groups, https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers

Reply:
How about that?


#pragma once

#include "EGL/egl.h"
#include "WinSystemX11.h"
#include "rendering/gles/RenderSystemGLES.h"

#include "platform/linux/OptionalsReg.h"

#include <memory>

Reply:
That is correct.